src/controllers/productController.js

```diff
@@ -1,14 +1,52 @@
-const { addProduct, getProducts } = require('../services/productService');
+const { addProduct, getProducts } = require("../services/productService");
 
-const getProductsEndpoint = async(req, res) => {
-    // TODO: Implement method
+const getProductsEndpoint = async (req, res) => {
+    const { success, products, error } = await getProducts();
+    if (success) {
+        res.status(200).json(products);
+    } else {
+        res.status(500).json({ error });
+    }
 };
 
-const addProductEndpoint = async(req, res) => {
-    // TODO: Implement method
+const addProductEndpoint = async (req, res) => {
+    const { description, full_description, category, price, restaurant_id, active } = req.body;
+    const { success, productId, error } = await addProduct(id, description, full_description, category, price, restaurant_id, active);
```

Comment:
Hi! In this case, we don't need to use the id.
```suggestion
    const { success, productId, error } = await addProduct(description, full_description, category, price, restaurant_id, active);
```